.idea/NetflixTest.iml

```diff
@@ -11,16 +11,22 @@
     <orderEntry type="library" scope="PROVIDED" name="builder (v3.2.3, ruby-2.6.5-p114) [gem]" level="application" />
```

Comment:
Files like this one should not be put up in git. To keep files you don't want in git out of git you use a file call .gitignore to exclude them. Don't worry about it here, I just wanted you to be aware of it.

Reply:
Thanks for pointing this out